desktop/funnelcake138/repack.cfg

```diff
@@ -0,0 +1,10 @@
+# funnelcake138
+locales="en-US"
+linux-i686=false
+linux-x86_64=false
+mac=false
+win32=false
```

Comment:
Are we sure we will never have a 32 bit download?

Reply:
The argument is there are much fewer win32 installs per day so it was more complexity for little gain.